### PR TITLE
security vulnerability patch

### PR DESCRIPTION
**Pull request by jjsjann123**

### CODE REVIEW

**Comment by jjsjann123** [comment]
!build

**Comment by jjsjann123** [comment]
Looks like CI is working.

File: .github/workflows/blossom-ci.yml [hunk 1]
```diff
@@ -24,8 +24,8 @@ jobs:
 
     # This job only runs for pull request comments
     if: |
-         contains( 'xwang233,jjsjann123,', format('{0},', github.actor)) &&
-         github.event.comment.body == '!dontusethis'
```
Comment (jjsjann123) [inline .github/workflows/blossom-ci.yml:28]: cc'ing @xwang233 are we still using this one? I thought it's just a toy example.

Can we remove it now?

Comment (xwang233) [reply .github/workflows/blossom-ci.yml:28]: will discuss this offline

**Review by xwang233 — APPROVED** [review]

**Pull request merged**